jdk/src/share/classes/java/security/Signature.java

```diff
@@ -584,6 +584,7 @@ public final void initVerify(Certificate certificate)
      * supposed to be used for digital signatures, an
      * {@code InvalidKeyException} is thrown.
      *
+     * @apiNote This method is defined in Java SE 8 Maintenance Release 3.
```

Comment:
Hmm, there are other pkg private methods added in the maintenance release which do not have the `@since` javadoc tag. For consistency sake, maybe removing the `@since` tag here and no need to add `@apiNote` then?

Reply:
I apparently let this slip through, this `@since` should not have been there.  Please remove it and the `@apiNote`.  Thanks.